src/App.jsx

```diff
@@ -54,19 +54,43 @@ function Todos() {
             'https://raw.githubusercontent.com/IdoGvili/TaskList-repo/master/src/Data.json',
         );
         const fetchedData = await response.json();
-        const newData = fetchedData.map((task) => ({ ...task, id: v4() }));
+        const newData = fetchedData.map((toDo) => ({ ...toDo, id: v4() }));
         setToDoList(newData);
     }, []);
     useEffect(() => {
         fetchData();
     }, [fetchData]);
-
+    const handleSearchTerm = useCallback((searchTermProp) => {
+        setSearchTerm(searchTermProp);
+    }, []);
     return (
         <>
-            <ToDoList toDoList={toDoList} onRemoveTodo={onRemoveTodo} />
+            <ToDoList
+                toDoList={toDoList}
+                onRemoveTodo={onRemoveTodo}
+                searchTerm={searchTerm}
+            />
             <ToDoForm onAddTodo={onAddTodo} />
+            <Search handleSearchTerm={handleSearchTerm} />
         </>
     );
 }
+function Search({ handleSearchTerm }) {
+    const handleChange = (e) => {
```

Comment:
`handleChange` does not imply anything. how are you handling the change? the idea is to give functions and variables descriptive names so that comments are not needed. describe what the function is should do...

let's remove this function. instead, pass a `onSearchTermChange` function as a prop and pass it to your input's onChange